packages/base-dracut-modules/50-cos-initrd.conf

```diff
@@ -2,4 +2,4 @@ hostonly_cmdline="no"
 hostonly="no"
 compress="xz"
 omit_dracutmodules+=" multipath "
-add_dracutmodules+=" livenet dmsquash-live "
+add_dracutmodules+=" fs-lib livenet dmsquash-live "
```

Comment:
mmm I am not so sure about it. If this is a dependency of `systemd-fsck` I doubt it should be here. Instead better add it in `depends()` in `packages/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh`, so each installation of the immutable-rootfs module will also pull and include this dracut module.